docs/archive/web_services/REST.md

```
@@ -15,31 +15,35 @@ have been defined.
 Use annotations in your web service implementation class to give details as to the URL, and the return type
 
+```java
 @Path("/providerService/")
 @Produces("application/xml")
 public class ProviderService {
 }
+```
 
 You can get an instance of the SecurityContext using
 
+	```java
```

markdownlint-cli2 (0.22.1)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)